src/main/java/com/nekomaster1000/infernalexp/client/entity/render/VolineGlowLayer.java

```diff
@@ -1,25 +1,42 @@
 package com.nekomaster1000.infernalexp.client.entity.render;
 
+import com.mojang.blaze3d.matrix.MatrixStack;
+import com.mojang.blaze3d.vertex.IVertexBuilder;
 import com.nekomaster1000.infernalexp.InfernalExpansion;
-import com.nekomaster1000.infernalexp.entities.EmbodyEntity;
-
+import com.nekomaster1000.infernalexp.entities.VolineEntity;
+import net.minecraft.client.renderer.IRenderTypeBuffer;
 import net.minecraft.client.renderer.RenderType;
 import net.minecraft.client.renderer.entity.IEntityRenderer;
 import net.minecraft.client.renderer.entity.layers.AbstractEyesLayer;
 import net.minecraft.client.renderer.entity.model.EntityModel;
+import net.minecraft.client.renderer.texture.OverlayTexture;
+import net.minecraft.entity.ai.attributes.Attributes;
 import net.minecraft.util.ResourceLocation;
 import net.minecraftforge.api.distmarker.Dist;
 import net.minecraftforge.api.distmarker.OnlyIn;
 
 @OnlyIn(Dist.CLIENT)
-public class VolineGlowLayer<T extends EmbodyEntity, M extends EntityModel<T>> extends AbstractEyesLayer<T, M> {
-    private static final RenderType RENDER_TYPE = RenderType.getEyes(new ResourceLocation(InfernalExpansion.MOD_ID,
-            "textures/entity/voline_glow.png"));
+public class VolineGlowLayer<T extends VolineEntity, M extends EntityModel<T>> extends AbstractEyesLayer<T, M> {
+
+    private static final RenderType RENDER_TYPE = RenderType.getEyes(new ResourceLocation(InfernalExpansion.MOD_ID, "textures/entity/voline_glow.png"));
+    private static final RenderType TIRED_RENDER_TYPE = RenderType.getEyes(new ResourceLocation(InfernalExpansion.MOD_ID, "textures/entity/voline_tired_glow.png"));
 
     public VolineGlowLayer(IEntityRenderer<T, M> rendererIn) {
         super(rendererIn);
     }
 
+    public RenderType getVolineRenderType(VolineEntity entity) {
+        return entity.getAttributeValue(Attributes.MOVEMENT_SPEED) <= 0 ? TIRED_RENDER_TYPE : RENDER_TYPE;
```

Comment:
How can movement speed be less than 0
```suggestion
        return entity.getAttributeValue(Attributes.MOVEMENT_SPEED) == 0 ? TIRED_RENDER_TYPE : RENDER_TYPE;
```

Reply:
it's just a precautionary. no harm in making it <= just in case it happens to be below zero